run_dir/design/worksets.html

```diff
@@ -48,16 +48,22 @@ <h1 id="{{ worksets }}" class="worksets_page_heading"><span id="page_title">All
                  <td>
                  {% for onepj in ws_data.get(onews).get(onekey[1]) %}
                  <abbr title="{{ ws_data.get(onews).get(onekey[1]).get(onepj).get('project_name')}}">
-                     <a href="/project/{{ onepj }}">{{ onepj }}</a></abbr>
+                     <a href="/project/{{ onepj }}">{{ ", ".join([ws_data.get(onews).get(onekey[1]).get(onepj).get('project_name'), onepj]) }}</a></abbr>
```

Comment:
Can you change this 
```
ws_data.get(onews).get(onekey[1]).get(onepj).get('project_name')
```
to
```
ws_data.get(onews, {}).get(onekey[1], {}).get(onepj, {}).get('project_name', 'NA')
```
So it wouldn't fail in case some entry miss the required stuff